cppwamp/include/cppwamp/udsconnector.hpp

```diff
@@ -20,6 +20,7 @@
 #include "connector.hpp"
 #include "error.hpp"
 #include "rawsockdefs.hpp"
+#include "./internal/udsopener.hpp"
```

Comment:
You're right. I didn't notice the parallel with `tcpconnector.hpp`. It's due to some inconsistency in how clang handles nested classes for forward declarations. I'll have a better fix shortly.